.travis.yml

```diff
@@ -2,31 +2,49 @@ language: cpp
 compiler:
     - gcc
     - clang
+env:
+    - TARGET=cpp
+    - PYVERSION=2.6.9
+    - PYVERSION=2.7.8
+    - PYVERSION=3.2.6
+    - PYVERSION=3.3.6
+    - PYVERSION=3.4.3
 
 before_install:
     - echo "yes" | sudo add-apt-repository ppa:kalakris/cmake
```

Comment:
Why do we need a specific version of cmake? Is this some trickery we should document here?